### PR TITLE
doc: chip: update name to matter

### PR DESCRIPTION
**Pull request by greg-fer**
Updated naming for former Project CHIP protocol in docs.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

File: samples/connectedhomeip/light_bulb/README.rst [hunk 1]
```diff
@@ -1,19 +1,20 @@
+.. _matter_light_bulb_sample:
 .. _chip_light_bulb_sample:
```
Comment (Damian-Nordic) [inline samples/connectedhomeip/light_bulb/README.rst:2]: Remove the old label?

File: samples/connectedhomeip/light_switch/README.rst [hunk 2]
```diff
@@ -1,14 +1,15 @@
+.. _matter_light_switch_sample:
 .. _chip_light_switch_sample:
```
Comment (Damian-Nordic) [inline samples/connectedhomeip/light_switch/README.rst:2]: same here?

File: samples/connectedhomeip/lock/README.rst [hunk 3]
```diff
@@ -1,14 +1,15 @@
+.. _matter_lock_sample:
 .. _chip_lock_sample:
```
Comment (Damian-Nordic) [inline samples/connectedhomeip/lock/README.rst:2]: same here?

**Review by Damian-Nordic — APPROVED** [review]

**Review by LuDuda — APPROVED** [review]

File: samples/connectedhomeip/light_bulb/README.rst (hunk 1 above)
Comment (greg-fer) [reply samples/connectedhomeip/light_bulb/README.rst:2]: Let's keep it until we completely switch to the new naming convention.

**Review by carlescufi — CHANGES_REQUESTED** [review]
What about west.yml and the repo name itself, should those change too?

**Comment by LuDuda** [comment]
@carlescufi the rebranding happened two days ago, we would like to wait and coordinate this effort with the community upstream (eg. on repo name).  We plan to make subsequent PR before the code freeze.

**Review by carlescufi — APPROVED** [review]

**Pull request merged**